Pipfile

```diff
@@ -21,6 +21,7 @@ boto3 = "*"
 watchtower = "*"
 insights-core = "*"
 bitmath = "*"
+app-common-python = {git = "https://github.com/RedHatInsights/app-common-python"}
```

Comment:
Do we really want to get that code from master all the time?

We don't have direct control over it so I suggest to do it by git tags.

Maybe we can ask how often `app-common-python` is tagged? (@psav?)

Reply:
@petrblaho I asked @jpramos123 about that, and he told me `app-common-python` doesn't use tag for while..  the last tag as in April, and there's no tag anymore...

Reply:
@fellipeh @petrblaho In a conversation with @psav he told me that we should use `pip install app-common-python` as it is in pip.